packages/l10n/package.json

```diff
@@ -44,8 +44,7 @@
     "@commercetools-frontend/sentry": "16.5.3",
     "@types/prop-types": "^15.7.3",
     "moment": "^2.24.0",
-    "prop-types": "15.7.2",
-    "react": "16.13.0"
```

Comment:
Moving this down into devDeps.

Reply:
Ah, it should probably also be a peer dep. Good catch!

Reply:
React is a devDep mostly. So I presume we add a peerDep requirement here too.

Reply:
2e2d854